CMakeLists.txt

```diff
@@ -69,31 +62,18 @@ else (BUILD_MANPAGES)
 endif (BUILD_MANPAGES)
 
 find_package(Boost 1.53.0 REQUIRED COMPONENTS program_options filesystem system )
```

Comment:
Do we need to add every boost header that we use here, because if that is the case this line is out of date?

Reply:
I think so

Reply:
@JoshuaSBrown, yes, we should add every package we are using here. Which ones are missing?